docker-for-ibm-cloud/quickstart.md

```diff
@@ -103,28 +104,21 @@ Congrats! Your Docker EE for IBM Cloud cluster is provisioning. First, the manag
 
 ## Step 4: Use UCP
 
-Check it out: Docker for IBM Cloud uses [Docker Universal Control Plane (UCP)](/datacenter/ucp/2.2/guides/) to help you manage your cluster through a simple web UI!
+Check it out: Docker for IBM Cloud uses [Docker Universal Control Plane (UCP)](/datacenter/ucp/2.2/guides/) to help you manage your cluster through a simple interface!
```

Comment:
In the EE docs we tend to use "web UI" instead of "interface", the rationale being that CLIs and APIs are also interfaces.